src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1461,9 +1461,12 @@ private function registerSecurityCsrfConfiguration(array $config, ContainerBuild
         }
     }
 
-    private function registerSerializerConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
+    private function registerSerializerConfiguration(array $config, ContainerBuilder $container, PhpFileLoader $loader)
     {
-        $loader->load('serializer.xml');
+        $loader->load('serializer.php');
+
+        $serializerExtractor = $container->getDefinition('property_info.serializer_extractor');
+        $serializerExtractor->setPublic(false);
```

Comment:
what is this for?

Reply:
Tests don't pass without it. 
![image](https://user-images.githubusercontent.com/6825062/84495339-ad030080-acb3-11ea-9fd9-62ac61aa8274.png)

Reply:
this really should not be needed as it wasn't needed before. you sure the test fails without it? then there is something else buggy.

Reply:
Yes, tests fail without it (proof on the screenshot above). May be I made something wrong in this task? But I don't see the error...
If bug exists, I don't know how to fix it.

Reply:
I can confirm the test fails. When using the php config, the service is suddenly public.

Reply:
By adding `$container->services()->defaults()->private()` the definitions are private which makes the tests pass. But we assumed this would not be needed anymore.

Reply:
Thanks for your help :)